packages/tupaia-web/src/features/Map/MapOverlaySelector/MobileMapOverlaySelector.tsx

```diff
@@ -44,40 +55,77 @@ const OverlayLibraryHeader = styled.span`
   width: 100%;
 `;
 
+const OverlayListWrapper = styled.div`
+  padding: 1rem;
+`;
+
 const OverlayMenu = styled.div<{
   $expanded: boolean;
 }>`
-  height: ${({ $expanded }) => ($expanded ? '100%' : '0')};
+  height: ${({ $expanded }) => ($expanded ? '95vh' : '0')};
```

Comment:
```suggestion
height: ${({ $expanded }) => ($expanded ? `calc(100vh - ${TOP_BAR_HEIGHT_MOBILE}px)` : '0')};
```
I think the 95vh is a magic number and not always going to fit the space. We could move the `TOP_BAR_HEIGHT_MOBILE` to the constants file and use it to calculate the height.